backend/data/static/datasets_raw.py

```diff
@@ -0,0 +1,86 @@
+import pandas as pd
```

Comment:
fab - this works so far and this is among the few I would retain in the `/backend/data/` folder - as the previous `datasets_clean`.

You can leave it also in `../static/` (as the other _clean one) if you like but I would also allow to import this module without going into `.static`.

Let me explain this better! So, having _importable_ Python modules in `/backend/data/` we officially promote this from a "normal" directory to a Python **package**.

This means that in `/backend/data/` folder there will also be a `__init__.py` file - which is the **main** file controlling the interface of the package. IOW, determining what you can or cannot import when you will simply do

(for example from a main script in `/backend/`)

```python
from data import ???
```

So, if in the `__init__.py` you would do just:
```python
from static.datasets_clean import *
from static.datasets_raw import *
```

You're basically promoting all the imported objects from those two modules to be in the main package namespace. Practically speaking this means that then you could do:

```python
from data import WELSH_LSOA
```

instead of 
```python
from data.static.datasets_raw import WELSH_LSOA
```

Easier and cleaner, isn't it?